test/functional/sqlsrv/MsHelper.inc

```diff
@@ -22,6 +22,9 @@ const INSERT_PREPARE = 2;
 const INSERT_QUERY_PARAMS = 3;
 const INSERT_PREPARE_PARAMS = 4;
 
+const KSP_NAME = 'MyCustomKSPName';
+const ENCRYPT_KEY = 'LPKCWVD07N3RG98J0MBLG4H2';
+const KSP_TEST_TABLE = 'CustomKSPTestTable';
```

Comment:
I'm not sure if these constants should be here. To me I think MsHelper should only contain helper functions and not any constants. Maybe define them in MsSetup?

Reply:
But they are AE related constants... the variables in MsSetup apply to all tests, right?

Reply:
these constants apply to all tests eventually since all tests are supposed to run in the ksp flavor at the end. This is just a suggestion, you may leave it here if you want.

Reply:
As I said before, these are AE related constants, so I will keep them here. Those in MsSetup are variables, meant to be changed based on test settings.